src/client/components/login/components/LoginForm.jsx

```diff
@@ -1,31 +1,42 @@
 import React, { PropTypes } from 'react'
 import { Field, reduxForm } from 'redux-form'
 import { onSubmitActions } from 'redux-form-submit-saga'
-import RaisedButton from 'material-ui/RaisedButton'
+import { RaisedButton } from 'material-ui'
+import { Card } from '../../card'
 import { Alert } from '../../alert'
 import { TextField } from '../../form'
+import { required } from '../../form/validators'
 
-export function LoginForm({ handleSubmit, pristine, submitting, error }) {
+export function LoginForm({ handleSubmit, submitting, valid, error }) {
   return (
-    <form onSubmit={handleSubmit}>
-      {error && <Alert message={error} type={'error'} />}
-      <Field name="email" component={TextField} label="Email" fullWidth />
-      <Field name="password" component={TextField} label="Wachtwoord" type="password" fullWidth />
-      <RaisedButton
-        type="submit"
-        label="Inloggen"
-        primary
-        fullWidth
-        disabled={pristine || submitting}
-      />
-    </form>
+    <Card padding="20" size={Card.SIZE.small} title="Login">
```

Comment:
I'd expect a unit for the padding here. Ah wait, react adds px automatically. In that case I'd use em or rem, since those are the units we use mostly for sizing.